content/kubeone/master/using_kubeone/static_workers.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Using Static Workers"
+date = 2020-04-15T12:00:00+02:00
++++
+
+Starting with version `v1.0.0`, KubeOne Added support for provisioning Kubernetes clusters with static worker nodes.
+
+This is useful in cases where the infrastructure provider is not supported by [machine-controller][1]. In this case, it's possible to disable machine-controller deployment and use the static worker nodes provided by the infrastructure provider.
```

Comment:
Please drop the "disable machine-controller" part, we want to remove possibility to not to deploy machine-controller for CSR approver controller reason.